easypost/__init__.py

```diff
@@ -240,7 +211,7 @@ def _encode_inner(cls, params):
                     pass
 
                 out.append((key, value))
-        return out
+        return urlencode(out)
```

Comment:
Could we assign `urlencode(out)` to a variable and return a clean variable?

Reply:
If you prefer. I tend to favor the opposite because I don't like extra indirection.